scripts/calendar/mongolian_generator.py

```
@@ -0,0 +1,204 @@
+
+
+def int_div(a, b):
+    return a // b
+
+
+def amod(a, b):
+    t = a % b
+    return t + b if t <= 0 else t
+
+
+Mzero = 3
```

Pylint (3.3.7)

[convention] Constant name "Mzero" doesn't conform to UPPER_CASE naming style

(C0103)